src/matgl/layers/_graph_convolution.py

```diff
@@ -243,12 +246,12 @@ def __init__(
         state_update_func: Module | None,
     ):
         """Parameters:
-        include_state (bool): Whether including state
```

Comment:
Again, you cannot change variable names on a whim. This can be done in a backward compatible manner. But more importantly, did you even update the version numbers?